src/components/Modals/ExploreModals/SearchModal/SearchModal.tsx

```diff
@@ -79,56 +177,113 @@ const SearchModal = ({ modalOpen, changeOpen }: ExploreModalProps) => {
           <Text style={{ fontSize: 20 }}>Accomodations</Text>
           <View style={{ width: 34 }} />
         </View>
-        <View>
-          <Collapsable
-            title="Where"
-            subtitle="Nowhere"
-            contentTitle="Pick your next destination"
-            collapsed={isCollapsed.where}
-            onTouch={toggleCollapseWhere}
-          >
-            <Input placeholder="Where are you going?" />
-          </Collapsable>
-          <Collapsable
-            title="When"
-            subtitle="Never"
-            contentTitle="What are the dates?"
-            collapsed={isCollapsed.when}
-            onTouch={toggleCollapseWhen}
-          >
-            <DatePicker
-              options={{
-                backgroundColor: colors ? BLACK : WHITE,
-                textHeaderColor: colors ? WHITE : BLACK,
-                textDefaultColor: colors ? WHITE : BLACK,
-                selectedTextColor: WHITE,
-                mainColor: TOMATO,
-                textSecondaryColor: GREY_200,
-                borderColor: colors ? BLACK : WHITE,
-              }}
-              mode="calendar"
-              startDate={startDate}
-              endDate={endDate}
-              onStartDateChange={(date: Date) => setStartDate(date)}
-              onEndDateChange={(date: Date) => setEndDate(date)}
-            />
-          </Collapsable>
-        </View>
-      </SafeAreaView>
+        <FlatList
+          data={sections}
+          keyExtractor={(item) => item.type}
+          renderItem={({ item }) => {
+            if (item.type === 'location') {
+              return (
+                <Collapsable
+                  title={item.title}
+                  subtitle={location ? location : "Anywhere"}
```

Comment:
you already have constant for 'Anywhere', so use it, just transver to constants file

Reply:
done